diff --git a/dist/gen/index.js b/dist/gen/index.js
index 500863734b896bafa4522c9d5f4500b7d9a3d883..67aa0fa2752cff2ab72175f7d5e145eccc3c5d0d 100644
--- a/dist/gen/index.js
+++ b/dist/gen/index.js
@@ -1,6 +1,6 @@
 Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });
-const require_script = require('../node_modules/typebox/build/type/script/script.js');
-require('../node_modules/typebox/build/type/index.js');
+const require_script = require('typebox/type');
+require('typebox/type');
 
 //#region src/gen/index.ts
 const matchRoute = /: Elysia<(.*)>/gs;
diff --git a/dist/gen/index.mjs b/dist/gen/index.mjs
index 27baa3f4900963856809d0b13dc49e91f454a1d3..936ed97fd99a5e3dec3a1fdd6512a0ade6e56071 100644
--- a/dist/gen/index.mjs
+++ b/dist/gen/index.mjs
@@ -1,5 +1,5 @@
-import { Script } from "../node_modules/typebox/build/type/script/script.mjs";
-import "../node_modules/typebox/build/type/index.mjs";
+import { Script } from "typebox/type";
+import "typebox/type";
 
 //#region src/gen/index.ts
 const matchRoute = /: Elysia<(.*)>/gs;
